### PR TITLE
doc-strings and file-tree

### PR DESCRIPTION
**Pull request by wesleyktatum**
- Updated doc-strings in function and class definitions to consistent format, in keeping with the Sphinx GitHub documentation template

- Changed file-tree to more consolidated and expandable structure

### CODE REVIEW

File: ssts/utils/seg_label_utils.py [hunk 1]
```diff
@@ -0,0 +1,350 @@
+import os
```
Comment (dtorrejo) [inline ssts/utils/seg_label_utils.py]: What are the methods in here ("seg_label_utils") used for?

File: ssts/utils/main_methods.py [hunk 2]
```diff
@@ -0,0 +1,887 @@
+import os
```
Comment (dtorrejo) [inline ssts/utils/main_methods.py]: Main methods is too generic. We could break up this file into 3 smaller ones: Noise removal (outlier extraction and frequency removal), correlation and distributions methods, and plotting functions.

**Comment by dtorrejo** [comment]
You have some conflicting files; you will have to pull the latest code from the repo and fix issues. I can help you with this if needed.

**Comment by dtorrejo** [comment]
Let's rename the jupyter notebooks with Diego and Anton's names in them with more descriptive names: Diego_updated could be something like "analysis_num_components" or something like that; and Anton could be the same without his name.

**Comment by dtorrejo** [comment]
Finally, are all these jupyter notebooks needed? If so, I don't have any problem keeping them. We may want to have two directories under scripts: one for experiments (like diego_updated) and one for sample code (like SSTS current capabilities and example). Let me know what you think

File: ssts/utils/seg_label_utils.py (hunk 1 above)
Comment (wesleyktatum) [reply ssts/utils/seg_label_utils.py]: I've added a descriptive, intro doc-string to the module

**Review by wesleyktatum — COMMENTED** [review]
I've added a .gitignore file, which includes many of these jupyter notebooks. I'll go through and rename the remaining ones to be more descriptive.

**Comment by wesleyktatum** [comment]
I've pushed some changes addressing the main_methods.py break-up, re-adddressing those functions and calls, and making sure my local segmentation modules reflect the ponl/SSTS fork

**Pull request merged**